stock_production_lot_active/tests/test_lot_active.py

```diff
@@ -8,6 +8,7 @@ class TestLotActive(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
@AEstLo FYI, the better is to use odoo `base` addon test class `BaseCommon` as this is already included in it.